nh_eobs_mobile/controllers/main.py

```diff
@@ -890,13 +892,8 @@ def location_not_false_or_already_exists():
             wardboard_records = obj_nh_clinical_wardboard.search(safe_eval(f.domain))
             if wardboard_records:
                 locations = set([l.ward_id.name for l in wardboard_records])
-                for location in locations:
-                    if location_not_false_or_already_exists():
-                        favourites.append({
-                            'location': location,
-                            'default': '{}'.format(f.is_default).lower(),
-                        })
-
+                favourites.extend([{'location': l, 'default': '{}'.format(f.is_default).lower()} for l in locations])
```

Comment:
```suggestion
                favourites.extend(
                    {'location': l, 'default': '{}'.format(f.is_default).lower()} for l in locations)
```

Break before the `for` too and line up with `{` if that takes it over 88 chars still.